src/saltext/salt_describe/modules/salt_describe_cron.py

```diff
@@ -0,0 +1,129 @@
+# Copyright 2023 VMware, Inc.
+# SPDX-License-Identifier: Apache-2.0
+"""
+Module for building state file
+
+.. versionadded:: 1.0
```

Comment:
I think it needs to be 1.1.0 since we are using symantec versioning and this is a backwards compatible feature.